maia/maia.html

```diff
@@ -1,8 +1,17 @@
 <html>
     <head>
         <title>Maia's Page - Hacktoberfest 2020</title>
+        <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
     </head>
     <body>
-        Let's do this!
+        <!-- Page Content -->
+        <div class="flex content-center justify-center h-screen" style="background-color: #87C090;">
+            <div class="flex flex-col justify-center content-center">
+                <span class="text-4xl text-gray-300">
+                    <strong>Hey!</strong> Let's do this!
+                </span>                
+            </div>
+        </div>
+        <p style="text-align: center;"><i>Wow I love Hacktoberfest.</i></p>
```

Comment:
Normally I'm super opposed to including styling as an attribute to the html tags, but since this exercise is to learn Git, I'm going to approve it.

If you need another Pull Request to get your 4 for Hacktoberfest, consider going through and cleaning up the CSS - You could even create a separate file and reference that if you really wanted to clean it up well!

Good work!